lib/search.php

```
@@ -0,0 +1,55 @@
+function zw_exclude_linked_fragments_from_search(WP_Query $query): void
+{
+    if (is_admin() || !$query->is_main_query() || !$query->is_search()) {
+        return;
+    }
+
+    $search = trim((string) $query->get('s'));
+    if ($search === '') {
+        return;
+    }
+
+    $articles = new WP_Query([
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '19', column '21'). (undefined)

(MissingImport)